packages/backend-core/src/features/index.ts

```diff
@@ -150,6 +156,26 @@ export class FlagSet<V extends Flag<any>, T extends { [key: string]: V }> {
         }
       }
 
+      const license = ctx?.user?.license
+      if (license) {
+        for (const feature of license.features) {
+          const flag = this.flags[feature]
+          if (!flag) {
+            continue
+          }
+
+          if (flags[feature] === true || specificallySetFalse.has(feature)) {
+            // If the flag is already set to through environment variables, we
+            // don't want to override it back to false here.
+            continue
+          }
+
+          // @ts-ignore
```

Comment:
NAB: linters prefer @ts-expect-error, ideally with a comment or ticket link for addressing in future.

Reply:
Good call, have updated all instances of this. It's all the same error, which I'm not sure there's a good workaround for sadly.